modules/peers.js

```diff
@@ -107,7 +125,7 @@ __private.getByFilter = function (filter, cb) {
 	};
 
 	// Apply filters (by AND)
-	var peers = library.logic.peers.list(true);
+	var peers = library.logic.peers.list();
```

Comment:
Yep, but from that list API response is generated, isn't it? So API response will contain that RPC property and not contains properties that `library.logic.peers.list(true);` is normalizing. Please check that.